docs/installing.rst

```diff
@@ -63,4 +63,16 @@ Note: The command to install may vary system to system. Try another if one fails
       .. code-block:: sh
 
           $ pip install "git+https://github.com/pincer-org/pincer"
+          
+Installing [speed]
+-------------------
+To Get Our [speed] Version with aiohttp speed up just do this
+
+.. tab-set::
+
+    .. tab-item:: Universal Download Command
+
+      .. code-block:: sh
+
+          $ python3 -m pip install pincer[speed] # try replacing `3` with the version you have
```

Comment:
remove this or change it to the standard pip command

Reply:
@Endercheif why? python3 is the standard path for all os's

Reply:
I was referring to use the standard `pip install pincer[speed]` command as it is the default.
Also, it does not need to be a tabbed item since it's only one option.